src/metabase/lib/js.cljs

```diff
@@ -1210,6 +1210,14 @@
   [arg]
   (and (map? arg) (= :metadata/column (:lib/type arg))))
 
+(defn ^:export is-metric-metadata
+  "Returns true if arg is an MLv2 metric, ie. has `:lib/type :metadata/metric`.
+
+  > **Code health:** Smelly. When is this called and why does the FE need to know? The values are supposed to be opaque,
```

Comment:
This is not smelly. This should be `Single use`, explaining that it is needed for the expression editor only. The same for `is-column-metadata`